### PR TITLE
thellimist/PI-678

### PR DESCRIPTION
**Pull request by thellimist**
updated ApplyDefault settings to inform the user with modals and notifications

### CODE REVIEW

File: src/js/actions/pluginSettings.js [hunk 1]
```diff
@@ -68,11 +68,19 @@ export function asyncPluginFetchSettings(zoneId) {
 export function asyncPluginUpdateSetting(settingName, zoneId, value) {
     return (dispatch, getState) => {
         let oldSetting = getState().pluginSettings.entities[zoneId][settingName];
+
+        if (settingName == "default_settings") {
```
Comment (jwineman) [inline src/js/actions/pluginSettings.js]: Lets use a loading spinner instead of an info notification here.

Comment (thellimist) [inline src/js/actions/pluginSettings.js]: Do you mean cf-component-loading?

Comment (jwineman) [inline src/js/actions/pluginSettings.js]: Yup, while its fetching display the loading spinner.

Comment (thellimist) [inline src/js/actions/pluginSettings.js]: `ApplyDefaultSettingsCard.js` calls `dispatch(asyncPluginUpdateSetting(SETTING_NAME, activeZoneId, VALUE));`

Is there an easy way to catch the result of `asyncPluginUpdateSetting` inside `ApplyDefaultSettingsCard`?

Comment (jwineman) [inline src/js/actions/pluginSettings.js]: The `_SUCCESS` and `_ERROR` options update `isFetching`.  You can tell whether the call has completed based on that.

File: lang/en.js [hunk 2]
```diff
@@ -45,6 +45,14 @@
     "container.applydefaultsettingscard.title": "Apply Default Settings",
     "container.applydefaultsettingscard.description": "We recommend you to use default settings",
     "container.applydefaultsettingscard.button": "Apply",
+    "container.applydefaultsettingscard.success": "Your settings has been successfully set.",
+    "container.applydefaultsettingscard.info": "This may take a while.",
```
Comment (jwineman) [inline lang/en.js]: Don't need this since we have the loading modal right?

File: lang/en.js [hunk 3]
```diff
@@ -45,6 +45,14 @@
     "container.applydefaultsettingscard.title": "Apply Default Settings",
     "container.applydefaultsettingscard.description": "We recommend you to use default settings",
     "container.applydefaultsettingscard.button": "Apply",
+    "container.applydefaultsettingscard.success": "Your settings has been successfully set.",
```
Comment (jwineman) [inline lang/en.js]: Lets make this say "Your default settings have been successfully set."

File: lang/en.js [hunk 4]
```diff
@@ -45,6 +45,14 @@
     "container.applydefaultsettingscard.title": "Apply Default Settings",
     "container.applydefaultsettingscard.description": "We recommend you to use default settings",
     "container.applydefaultsettingscard.button": "Apply",
+    "container.applydefaultsettingscard.success": "Your default settings have been successfully set.",
+    "container.applydefaultsettingscard.info": "This may take a while.",
```
Comment (jwineman) [inline lang/en.js]: Don't need this anymore.

**Pull request merged**